ml-agents/mlagents/serialization.py

```diff
@@ -0,0 +1,205 @@
+import logging
```

Comment:
would it make sense to call this file `onnx_serialization.py`?

Reply:
It also handles the tensorflow_to_barracuda conversion (those methods were migrated out of `tf_policy.py`). Maybe`model_serialization.py` instead?

Reply:
eh, guess not. looks like a lot more is going on here.

Reply:
i like `model_serialization.py`